tests/test_extra.lua

```diff
@@ -2463,6 +2463,28 @@ T['pickers']['history()']['validates arguments'] = function()
   validate({ scope = '1' }, '`pickers%.history`.*"scope".*"1".*one of')
 end
 
+T['pickers']['history()']['has custom "edit_command" mapping'] = function()
+  child.set_size(20, 70)
+
+  pick_history()
+  eq(child.lua_get('MiniPick.get_picker_opts().mappings.edit_command.char'), '<C-e>')
+
+  type_keys(':', '<C-e>')
+  eq(child.fn.getcmdtype(), ':')
+  eq(child.fn.getcmdline(), 'lua _G.n = _G.n + 2')
+
+  type_keys('<C-c>')
+  pick_history()
+  type_keys('/', '<C-e>')
+  eq(child.fn.getcmdtype(), '/')
+  eq(child.fn.getcmdline(), 'bbb')
+
+  type_keys('<C-c>')
+  pick_history()
+  type_keys('@', '<C-e>')
+  child.expect_screenshot({})
```

Comment:
No need for screenshot testing here. Just testing that command line is empty and the picker is still active should be enough.